src/pages/InfoPage/infoPage.module.scss

```diff
@@ -0,0 +1,68 @@
+.userOut{
+    padding: 180px 40px;
+    width: 100%;
+    background-color: rgb(153, 232, 158);
+}
+
+.userIs{
+    padding: 180px 40px;
+    width: 100%;
+    background-color: rgb(255, 231, 72);
+}
+.BtnOut{
```

Comment:
a lot of style duplicates. Create one general style and call it like btn. And then add aditional styles for that main className